packages/sitecore-jss-nextjs/src/components/NextImage.test.tsx

```diff
@@ -0,0 +1,232 @@
+/* eslint-disable no-unused-expressions */
+import chai from 'chai';
+import chaiString from 'chai-string';
+import { mount } from 'enzyme';
+import React from 'react';
+import { imageField as eeImageData } from '../testData/ee-data';
+import { NextImage, ImageField, loader } from './NextImage';
+
+const expect = chai.use(chaiString).expect;
+
+describe('<NextImage />', () => {
+  describe('Next loader function', () => {
+    const props = {
+      src: '/assets/img/test0.png',
+      width: 100,
+      height: 50,
+    };
+    const result = loader(props);
+
+    it('should return url and set it to src', () => {
+      expect(result).to.be.a('string');
+      expect(result).to.equal('https://cm.jss.localhost/assets/img/test0.png?mw=100');
+    });
+  });
+
+  // TODO: test error cases
+  describe('error cases', () => {
+    const props = {
+      src: '/assets/img/test0.png',
+      loader: () => new URL('test').href,
+    };
+
+    it('should throw an error if src is present', () => {
+      expect(() => mount(<NextImage {...props} />)).throws(
+        'Detected conflicting props src or loader. If you wish to use these props, use next/image directly.'
+      );
+    });
+
+    it('should throw an error if loader is present', () => {
+      expect(() => mount(<NextImage {...props} />)).to.have.length(0);
+      expect(() => mount(<NextImage {...props} />)).throws(
+        'Detected conflicting props src or loader. If you wish to use these props, use next/image directly.'
+      );
+    });
+
+    describe('with direct image object, no value/editable', () => {
+      const props = {
+        field: {
+          src: '/assets/img/test0.png',
+        },
+        width: 8,
+        height: 10,
+        id: 'some-id',
+        className: 'w-100',
+      };
+      const rendered = mount(<NextImage {...props} />)
+        .find('Image')
+        .at(1);
+
+      it('should render Next/Image with url', () => {
```

Comment:
you can remove `Next/Image` it's understandable from the `describe` that we are testing `NextImage`